server/queues/tasks/ExportTask.ts

```diff
@@ -93,6 +95,12 @@ export default abstract class ExportTask extends BaseTask<Props> {
         }).schedule();
       }
       throw error;
+    } finally {
+      if (filePath) {
+        await fs.promises.unlink(filePath).catch((error) => {
```

Comment:
```suggestion
        void fs.promises.unlink(filePath).catch((error) => {
```

Probably no benefit to waiting here